src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -90,14 +90,17 @@ public partial class JsonArray : System.Text.Json.JsonNode, System.Collections.G
         void System.Collections.Generic.ICollection<System.Text.Json.JsonNode>.CopyTo(System.Text.Json.JsonNode[] array, int arrayIndex) { }
         System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
     }
-    public partial class JsonBoolean : System.Text.Json.JsonNode
+    public partial class JsonBoolean : System.Text.Json.JsonNode, System.IEquatable<System.Text.Json.JsonBoolean>
```

Comment:
I suppose these you changed only because you were already changing JsonNumber. For the main fork PR only send JsonNumber class, which I suppose is already your plan, just wanted to reiterate :)